From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:58:42 +0200
Subject: dns: send IPv6 connectivity probes to RIPE DNS

Short of disabling the code (we may do that in a subsequent patch
anyway), avoid sending to Google.

Increase probe period from 1s to 15s

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 net/dns/host_resolver_manager.cc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/net/dns/host_resolver_manager.cc b/net/dns/host_resolver_manager.cc
--- a/net/dns/host_resolver_manager.cc
+++ b/net/dns/host_resolver_manager.cc
@@ -151,12 +151,12 @@ const unsigned kMinimumTTLSeconds = kCacheEntryTTLSeconds;
 
 // Time between IPv6 probes, i.e. for how long results of each IPv6 probe are
 // cached.
-const int kIPv6ProbePeriodMs = 1000;
+const int kIPv6ProbePeriodMs = 15000;
 
-// Google DNS address used for IPv6 probes.
-const uint8_t kIPv6ProbeAddress[] = {0x20, 0x01, 0x48, 0x60, 0x48, 0x60,
-                                     0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                                     0x00, 0x00, 0x88, 0x88};
+// RIPE NCC k.root-servers.net. 2001:7fd::1 (anycasted)
+const uint8_t kIPv6ProbeAddress[] =
+    { 0x20, 0x01, 0x07, 0xfd, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01 };
 
 // ICANN uses this localhost address to indicate a name collision.
 //
--
2.25.1
